ql/src/java/org/apache/hadoop/hive/ql/DriverFactory.java

```diff
@@ -64,6 +65,9 @@ private static IReExecutionPlugin buildReExecPlugin(String name) throws RuntimeE
     if (name.equals("reoptimize")) {
       return new ReOptimizePlugin();
     }
+    if(name.equals("reexecutelostam")) {
```

Comment:
Could you please describe it briefly this in `ConfVars.HIVE_QUERY_REEXECUTION_STRATEGIES`?
...and I think you may enable this plugin by default